public/third-party-cookies-check/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Third Party Cookies Test</title>
+  </head>
+
+  <body>
+    <script>
+      const checkCookiesEnable = () => {
+        let isCookieEnabled = window.navigator.cookieEnabled ? true : false
+        if (typeof window.navigator.cookieEnabled == 'undefined' && !isCookieEnabled) {
```

Comment:
Why not just `window.navigator.cookieEnabled === undefined`?

Reply:
yeah, that's better